src/Header.vue

```diff
@@ -30,9 +30,15 @@ export default {
     }
   },
   methods: {
-    findPicture(date) {
-      this.$emit('submitted-date', date)
-      router.push({path:`/date/${date}`})
+    findPicture(event, date) {
+      event.preventDefault();
```

Comment:
What default event is it preventing here?

Reply:
It was defaulting to the current day's POTD. I'm not really sure why, but it kept going back to it.